### ISSUE TITLE
I can't start the server

### ISSUE THREAD

**Issue by sbrocos** [json-schema-faker/json-schema-server#16]
When I run 
`json-schema-server json_server/ -p 5000`
Ever show:
`env: node\r: No such file or directory`

the installation result:

```
npm install -g json-schema-server
/usr/local/bin/json-schema-server -> /usr/local/lib/node_modules/json-schema-server/bin/json-schema-server
/usr/local/lib
└── json-schema-server@0.3.1
```

In OSX

**Comment by pateketrueke** [comment]
I encourage you to use NVM instead of global NodeJS installations.

I'm using OSX too but I never installed NodeJS the way you did... I'm not sure, but it can help.

**Comment by robeady** [comment]
Same problem here. I believe it occurs because your bin script (bin/json-schema-server) has windows line endings.

Workaround: you can run the server like `node path/to/json-schema-server` (replace with the actual path, I haven't installed globally so I don't know if `node json-schema-server` works)

**Comment by pateketrueke** [comment]
I just tried on node v4.8.x and works fine, but on my node v6.10.x/v8.6.x setup I got the same error.

I'll investigate asap, thanks for your feedback!

**Comment by pateketrueke** [comment]
Hi, I just released `v0.4.0` hoping this is solved now.
